src/Symfony/Component/Notifier/Bridge/Firebase/FirebaseOptions.php

```diff
@@ -29,25 +29,28 @@ abstract class FirebaseOptions implements MessageOptionsInterface
 
     private array $data;
 
-    public function __construct(string $to, array $options, array $data = [])
+    private bool $useTopic;
+
+    public function __construct(string $tokenOrTopic, array $options, array $data = [], bool $useTopic = false)
```

Comment:
would it make sense to parse `$tokenOrTopic` for a string like `/topic/...` and add a deprecation message (but handle the flag correctly)?

Reply:
If set to "$useTopic = true" no deprecation message will be needed. The default topic is used in the previous version.